docs/notebooks/unequal_schema.ipynb

```diff
@@ -0,0 +1,273 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Unequal schema problems\n",
+    "\n",
+    "There are a few ways in which parquet files written with slightly different schema can cause issues in the import pipeline. They have a similar correction mechanism, so we discuss both here."
```

Comment:
Maybe we could add a bit more motivation here? "For example, most of the input files have no missed data, but some have it"

Reply:
I originally had a lot more here, but thought it would be too much read =]

Added back in some visuals and explanatory text about mismatched types.